openapi/karbon/types.bal

```diff
@@ -19,369 +19,369 @@ public type KarbonApiTimeV3TimesheetdtoArr KarbonApiTimeV3Timesheetdto[];
 public type KarbonApiTenantUserdtoArr KarbonApiTenantUserdto[];
 
 public type KarbonApiTimeV3Aggregatetimeentrydto record {
-    string? TimeEntryKey;
-    string? EntityKey;
-    string? WorkItemKey?;
-    string? ClientKey?;
-    string? ClientType?;
-    string? RoleName?;
-    string? TaskTypeName?;
-    int? Minutes;
-    float? HourlyRate?;
-    string[]? Descriptions?;
+    string TimeEntryKey;
```

Comment:
Please re-generate using `--nullable`

Reply:
Fixed in https://github.com/ballerina-platform/openapi-connectors/commit/16c596b9e3d3e67b08f2aa34441fb638b71a8647